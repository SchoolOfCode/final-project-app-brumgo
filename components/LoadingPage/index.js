import React, { Component } from "react";
import { StyleSheet, View, Image } from "react-native";
import { Button, Icon } from "react-native-elements";

export default function LoadingPage(props) {
  return (
    <View style={styles.container}>
      <Image
        style={styles.logo}
        source={require("../LoadingPage/brumGoLogoRed.png")}
      />

      <View style={styles.buttonGroup}>
        <Button
          title="Log In"
          titleStyle={{ fontSize: 18, color: "#E12B38" }}
          buttonStyle={{ borderRadius: 10, backgroundColor: "#FFFFFF" }}
          containerStyle={{ width: 300, height: 50, borderRadius: 10 }}
          // onPress={}
        />
        <Button
          title=" Guest"
          titleStyle={{ fontSize: 18, color: "#E12B38" }}
          buttonStyle={{ borderRadius: 10, backgroundColor: "#FFFFFF" }}
          containerStyle={{ width: 300, height: 70, borderRadius: 10 }}
          onPress={() => props.navigation.push("Main")}
        />
      </View>

      <View style={styles.storybook}>
        <Button
          icon={<Icon name="cog" type="font-awesome" color="#E12B38" />}
          titleStyle={{ fontSize: 18, color: "#E12B38" }}
          buttonStyle={{ borderRadius: 60, backgroundColor: "#FFFFFF" }}
          containerStyle={{ width: 50, height: 50, borderRadius: 10 }}
          onPress={() => props.navigation.push("StoryBook")}
        />
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#E12B38"
  },
  logo: {
    width: 150,
    height: 150
  },
  buttonGroup: {}
});
