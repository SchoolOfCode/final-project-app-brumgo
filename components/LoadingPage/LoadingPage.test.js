import React from "react";
import * as renderer from "react-test-renderer";
import LoadingPage from "./index";

describe("LoadingPage", () => {
  it("should match to snapshot", () => {
    const component = renderer.create(<LoadingPage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
