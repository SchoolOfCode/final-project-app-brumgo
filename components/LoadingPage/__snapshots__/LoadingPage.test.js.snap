// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E12B38",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../components/LoadingPage/brumGoLogoRed.png",
      }
    }
    style={
      Object {
        "height": 150,
        "width": 150,
      }
    }
  />
  <View
    style={Object {}}
  >
    <View
      style={
        Object {
          "borderRadius": 10,
          "height": 50,
          "width": 300,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#2089dc",
              "borderRadius": 10,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#E12B38",
                "fontSize": 18,
                "paddingBottom": 1,
                "paddingTop": 2,
                "textAlign": "center",
              }
            }
          >
            Log In
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "borderRadius": 10,
          "height": 70,
          "width": 300,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#2089dc",
              "borderRadius": 10,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#E12B38",
                "fontSize": 18,
                "paddingBottom": 1,
                "paddingTop": 2,
                "textAlign": "center",
              }
            }
          >
             Guest
          </Text>
        </View>
      </View>
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "borderRadius": 10,
          "height": 50,
          "width": 50,
        }
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#FFFFFF",
              "borderColor": "#2089dc",
              "borderRadius": 60,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "transparent",
                  "justifyContent": "center",
                }
              }
              theme={
                Object {
                  "colors": Object {
                    "disabled": "hsl(208, 8%, 90%)",
                    "divider": "#bcbbc1",
                    "error": "#ff190c",
                    "grey0": "#393e42",
                    "grey1": "#43484d",
                    "grey2": "#5e6977",
                    "grey3": "#86939e",
                    "grey4": "#bdc6cf",
                    "grey5": "#e1e8ee",
                    "greyOutline": "#bbb",
                    "platform": Object {
                      "android": Object {
                        "error": "#f44336",
                        "primary": "#2196f3",
                        "secondary": "#9C27B0",
                        "success": "#4caf50",
                        "warning": "#ffeb3b",
                      },
                      "ios": Object {
                        "error": "#ff3b30",
                        "primary": "#007aff",
                        "secondary": "#5856d6",
                        "success": "#4cd964",
                        "warning": "#ffcc00",
                      },
                    },
                    "primary": "#2089dc",
                    "searchBg": "#303337",
                    "secondary": "#8F0CE8",
                    "success": "#52c41a",
                    "warning": "#faad14",
                  },
                }
              }
              underlayColor="white"
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#E12B38",
                      "fontSize": 24,
                    },
                    Object {
                      "backgroundColor": "transparent",
                    },
                    Object {
                      "fontFamily": "FontAwesome",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                ï€“
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
